SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -32,12 +32,15 @@ const FString SchemaCompilerExe = CreateExePath(GDKProgramPath, TEXT("schema_com
 const FString SpatialOSDirectory = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial/")));
 const FString SpatialOSConfigFileName = TEXT("spatialos.json");
 const FString ChinaEnvironmentArgument = TEXT(" --environment=cn-production");
+const FString RuntimeExe = TEXT("runtime.exe");
+const FString InspectorExe = TEXT("inspector.exe");
 
 const FString SpatialOSRuntimePinnedStandardVersion = TEXT("0.5.1");
 const FString SpatialOSRuntimePinnedCompatbilityModeVersion = TEXT("14.5.4");
 
 const FString InspectorURL = TEXT("http://localhost:31000/inspector");
 const FString InspectorV2URL = TEXT("http://localhost:31000/inspector-v2");
+const FString SpatialOSInspectorPinnedStandardVersion = TEXT("1.1.2");
```

Comment:
There's no need for `Standard` in here. There are no variants for Inspector like there was with runtime ^^  (the runtime variants are being removed too)